exec/java-exec/src/main/java/org/apache/drill/exec/rpc/user/UserServer.java

```diff
@@ -91,6 +91,34 @@
   static {
     userConnectionMap = new ConcurrentHashMap<>();
   }
+  public static  String safeLogString(UserToBitHandshake inbound) {
```

Comment:
1. Please remove one space -> `static  String`/
2. Can this method be just private? Not public static? If yes, please move it to the end of the class.
3. Please add javadoc to the method.
4. Please consider method renaming to depict actual work it does.